### ISSUE TITLE
Try adding mask loss later in the training after some epoch, say 50

### ISSUE THREAD

**Issue by Ghostvv** [RasaHQ/rasa#7310]
Our mask loss might be too destructive initially. It might be a good idea to turn it on later after the attention is more or less trained to attempt to break a little bit the overfitting to particular tokens

**Comment by reprabhala** [comment]
@Ghostvv can I pick this up?

**Comment by Ghostvv** [comment]
@reprabhala thank you for the interest, but this issue requires updates to the tensorflow source code of the DIET classifier. I'd recommend you to pick another issue. We have a board where these issues are collected: https://github.com/orgs/RasaHQ/projects/23

**Comment by reprabhala** [comment]
Thank you @Ghostvv! Will look into it.
